### ISSUE TITLE
Ender Fluid Conduits are permanently in Round Robin mode

### ISSUE THREAD

**Issue by vdrake77** [SleepyTrousers/EnderIO#4774]
#### Issue Description:
Enderio Ender Fluid Conduits are in round robin mode and can't have that feature turned off

#### What happens:
If you set Enderio Fluid conduits to disable round robin, it goes back to enabled as soon as you leave.

#### What you expected to happen:
Put fluid the tanks according to priority

#### Steps to reproduce:

1.Open conduit
2.Click to disable round robin
3.Close conduit gui
4.Open it up again
5.Round Robin is enabled again
...

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: 5.0.29
- EnderCore: 5.35
- Minecraft: 1.12.2
- Forge: 2730

**Comment by HenryLoenwind** [comment]
@EpicSquid fixed?

**Comment by MatthiasMann** [comment]
This still seems to happen with EnderIO-1.12.2-5.0.38.

I'm extracting from IC2 solar distillers and (round robin is disabled on all extraction points for the 'brown' channel) inserting into an EnderIO tank (16 bucket capacity - about ~9 buckets in it) with priority 1 and into an IC2 canning machine with priority -1 - and both the tank and the canning machine gets filled.

**Comment by MatthiasMann** [comment]
@HenryLoenwind I wasn't aware that the EnderIO tank had an insert limit - I know it has a limit for how fast it will pull/push on it's own. But then again the solar distillers only produce ~1mB/second (ie really slow) - and with ~20 of these the inserts I saw was not higher than 10mB/t. These distillers have their own tick schedule based on when they have been placed. Also when I disconnected the canning machine all fluid was successfully inserted into the tank.
